src/pages/ProductListPage/ProductListPage.js

```diff
@@ -0,0 +1,44 @@
+import './ProductListPage.css';
+import Sidebar from "../../components/Sidebar/Sidebar";
+import ProductsList from "../../components/ProductsList/ProductsList";
+import {results as data} from '../../utils/mocks/en-us/products.json'
+import Footer from "../../components/Footer/Footer";
+import { useState, useEffect} from "react";
+
+const ProductListPage = () => {
+  const [filteredProducts, setFilteredProducts] = useState(data);
+  const [activeCategories, setActiveCategories] = useState([]);
+
+  useEffect(() => {
+    window.scrollTo(0, 0)
```

Comment:
Nice to see that you care about this kind of details.
Tip: try this `    window.scrollTo({behavior:'smooth',top:0})` it looks cool